qqqwww123cdv/hometask-3/rainfall.rb

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+def getting_data
```

Comment:
the method name should be imperative, so let it be just `get_data`

Reply:
robocop swears at "get_date"

Reply:
Naming/AccessorMethodName: Do not prefix reader method names with get_.
def get_data
    ^^^^^^^^